question28/js/password.js

```diff
@@ -0,0 +1,35 @@
+import { validateInputValue } from "./modules/validations";
+
+const submitButton = document.getElementById('js-submit-button');
+const passwordInputArea = document.querySelector(".js-password");
+const confirmPasswordInputArea = document.querySelector(".js-confirm-password");
+
+
+const checkAllValidity = () => {
+  return document.getElementsByTagName("input").length === document.getElementsByClassName("valid").length && passwordInputArea.value === confirmPasswordInputArea.value;
+}
+
+const toggleDisabledOfSubmitButton = (isValid) => {
+  submitButton.disabled = !isValid;
+}
+
+const showErrorMessageInputValuesMismatch = () => confirmPasswordInputArea.nextElementSibling.textContent = passwordInputArea.value !== confirmPasswordInputArea.value ? "入力されたパスワードが一致しませんでした" : "";
```

Comment:
nits
I thought `passwordInputArea.value ! == confirmPasswordInputArea.value` could be put together, but what do you think?
```
    if (passwordInputArea.value && confirmPasswordInputArea.value) {
      showErrorMessageInputValuesMismatch(); 
    }
```

```JavaScript
    if (passwordInputArea.value && confirmPasswordInputArea.value && passwordInputArea.value !== confirmPasswordInputArea.value) {
      confirmPasswordInputArea.nextElementSibling.textContent = "入力されたパスワードが一致しませんでした";
    }else{
      confirmPasswordInputArea.nextElementSibling.textContent = "";
   }
```

Reply:
Thank you for your review!
You are right. I have tried to reflect your opinion and write it in one condition. 
When there is no problem with validation, the error statement needs to be removed, so I have added that as well.
The validation here is a bit complicated and troubling part...so if you have any more appropriate ideas would be appreciated:bow:

fixed commit : https://github.com/csakyo/homework/pull/43/commits/709a5fea5cb2d2dc8d1756a6e7fc0bcbac05f0eb